ansible/lemmy.yml

```diff
@@ -35,6 +35,7 @@
       with_items:
         - { src: 'templates/docker-compose.yml', dest: '/lemmy/docker-compose.yml', mode: '0600' }
         - { src: 'templates/nginx.conf', dest: '/etc/nginx/sites-enabled/lemmy.conf', mode: '0644' }
+        - { src: '../docker/iframely.config.local.js', dest: '/lemmy/iframely.config.local.js', mode: '0600' }
```

Comment:
If you deploy this file with Ansible, you should use variables to configure it automatically (eg `{{ domain }}`). Otherwise people have to edit it manually, and then int will be overwritten the next time they run Ansible. Another option is to set `force=no` like with the Lemmy config.

Reply:
At least with that iframely file, you don't need to provide the domain, since the nginx sets it at `your_domain.com/iframely/` . If we find some things that need to be customized, we can add those at some point and put that in the templates folder.

Reply:
Ah okay, I thought the domain was required because I saw it in the config file. Maybe remove that line then?

Reply:
Its okay as is, its just iframely's default config, looks like the next line informs it to not use that anyway.